### ISSUE TITLE
[4.0] calendar field [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#29940]
If you try to use the calendar without using the mouse there are multiple issues

- it is not possible to navigate from the days to the time
- the time is using an input of select without a corresponding label
- if you are somehow able to select either the hour or minute then if you use the keyboard to change its value the date also changes
- i suspect that time should be a separate table and that would help solve the above
- the absence of any `th` or scope in the `thead` makes it impossible to know that 23rd sept is a wednesday
- the same is true for the `day wn` cell which should be a th with a scope of row so that you know it is the third week
- it is not possible to navigate to any of the three buttons using a keyboard
- it is not possible to navigate to the area to change the month or year using a keyboard

![cal](https://user-images.githubusercontent.com/1296369/86496203-29997400-bd74-11ea-8527-d4d6f9c4bd4a.gif)

**Comment by Quy** [comment]
Related #24019
